openhtf/output/web_gui/__init__.py

```diff
@@ -438,6 +482,10 @@ def __init__(self, discovery_interval_s, disable_discovery, http_port,
         (r'/', self.MainHandler, {'port': http_port}),
         (r'/station/(?:\d{1,3}\.){3}\d{1,3}/(?:\d{1,5})/?',
          self.MainHandler, {'port': http_port}),
+        (r'/phases/([\d\.]+)/(\d+)/(.*)/?', PhasesHandler,
```

Comment:
shouldn't this be `/station/(host)/(port)/(test_uid)/phases`? plugs is similarly wrong to me (but we can fix that later)